### DISCUSSION TITLE
Settings for mac mini running Monterey

### DISCUSSION THREAD

**Discussion by mbierman** [AlexGustafsson/homebridge-wol#211] [General]
### General idea

I am having trouble with finding the settings that work with a mac mini running Monterrey. For example, 
```
bigmac:~ michael$ pmset -g powerstate IODisplayWrangler

      Driver ID  Current State  Max State  Current State Description
Internal failure: Failed to get power state information
bigmac:~ michael$ pmset -g powerstate IODisplayWrangler
bigmac:~ michael$ pmset -g powerstate IODisplayWrangler 

      Driver ID  Current State  Max State  Current State Description
Internal failure: Failed to get power state information
```
I understand this isn't strictly speaking a bug in the plugin or anything but a more robust set of examples for popular platforms would be super helpful.  

### Use case

Can't use the plugin without fighuring out what the necessary commands are.

**Comment by AlexGustafsson** [comment]
Some previous discussion here: https://github.com/AlexGustafsson/homebridge-wol/discussions/178.

There is an example in the above thread that seems to be working.

**Reply by holmstockd** [reply]
did anyone give a good example i can not figure out that command on post 178.

**Reply by idnamrk** [reply]
> did anyone give a good example i can not figure out that command on post 178.

Here: https://github.com/AlexGustafsson/homebridge-wol/discussions/227#discussion-4818655
